src/librepgp/stream-parse.cpp

```diff
@@ -2573,19 +2583,22 @@ process_pgp_source(pgp_parse_handler_t *handler, pgp_source_t &src)
         src_close(&datasrc);
     } else {
         if (handler->ctx->detached) {
-            RNP_LOG("Detached signature expected.");
+            RNP_LOG("Attached signature expected.");
```

Comment:
> -Detached
> +Attached


is this correct?

Reply:
@antonsviridenko Yeah, some sort of double-check with typo from early times. Noticed when worked on surrounding code.